src/Npgsql/NpgsqlRawCopyStream.cs

```diff
@@ -186,7 +208,14 @@ public override int Read(byte[] buffer, int offset, int count)
         /// <summary>
         /// Cancels and terminates an ongoing operation. Any data already written will be discarded.
         /// </summary>
-        public void Cancel()
+        public void Cancel() => Cancel(false).GetAwaiter().GetResult();
+
+        /// <summary>
+        /// Cancels and terminates an ongoing operation. Any data already written will be discarded.
+        /// </summary>
+        public Task CancelAsync() => Cancel(true);
```

Comment:
No cancellation token needed here, but we still need NoSynchronizationContext missing.